addons/packages/kpack/0.5.0/README.md

```diff
@@ -0,0 +1,565 @@
+
+   Your secret create should look something like this:
+
+    ```bash
+    kp secret save tutorial-registry-credentials \
+       --registry https://index.docker.io/v1/ \
+       --registry-user my-dockerhub-username \
+       -n default
+    ```
+
+   > Note: Learn more about kpack secrets with the [kpack secret documentation](https://github.com/pivotal/kpack/blob/main/docs/secrets.mdd)
+
+
+6. Create a kpack image resource
+
+   An image resource is the specification for an OCI image that kpack should build and manage.
+
+   We will create a sample image resource that builds with the builder created in step five.
+
+   The example included here utilizes
+   the [Spring Pet Clinic sample app](https://github.com/spring-projects/spring-petclinic). We encourage you to
+   substitute it with your own application.
+
+   Create an image resource:
+
+    ```yaml
+    kp image save tutorial-image \
+      --tag <IMAGE-TAG> \
+      --git https://github.com/spring-projects/spring-petclinic \
+      --git-revision 82cb521d636b282340378d80a6307a08e3d4a4c4 \
+      --builder my-builder \
+      -n default
+    ```
+
+    - Make sure to replace `<IMAGE-TAG>` with the tag in the registry of the secret you configured in step #5. Something like:
+      `your-name/app` or `gcr.io/your-project/app`
+    - If you are using your application source, replace `--git` & `--git-revision`.
+   > Note: To use a private git repo follow the instructions in [secrets](https://github.com/pivotal/kpack/blob/main/docs/secrets.md)
+
+   You can now check the status of the image resource.
+
+   ```bash
+   kp image status tutorial-image -n default
+   ```
+
+   You should see that the image resource has a status Building as it is currently building.
+
+    ```
+    Status:         Building
+    Message:        --
+    LatestImage:    --
+    
+    Source
+    Type:        GitUrl
+    Url:         https://github.com/spring-projects/spring-petclinic
+    Revision:    82cb521d636b282340378d80a6307a08e3d4a4c4
+    
+    Builder Ref
+    Name:    base
+    Kind:    Builder
+    
+    Last Successful Build
+    Id:              --
+    Build Reason:    --
+    
+    Last Failed Build
+    Id:              --
+    Build Reason:    --
+    ```
+
+   You can tail the logs for image resource that is currently building using
+   the [kp cli](https://github.com/vmware-tanzu/kpack-cli/blob/main/docs/kp_build_logs.md)
+
+    ```
+    kp build logs tutorial-image -n default
+    ``` 
+
+   Once the image resource finishes building you can get the fully resolved built OCI image with `kubectl get`
+
+    ```bash
+    kp image status tutorial-image -n default
```

Comment:
the previous sentence refers to a kubectl command - is this meant to be kubectl command?

Reply:
The previous sentence was incorrect